src/LiveComponent/composer.json

```diff
@@ -26,27 +26,30 @@
         }
     },
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.1",
         "symfony/property-access": "^5.4.5|^6.0",
         "symfony/serializer": "^5.4|^6.0",
         "symfony/ux-twig-component": "^2.8"
     },
     "require-dev": {
         "doctrine/annotations": "^1.0",
-        "doctrine/doctrine-bundle": "^2.0",
-        "doctrine/orm": "^2.7",
+        "doctrine/collections": "^1.6.8|^2.0",
+        "doctrine/doctrine-bundle": "^2.4.3",
+        "doctrine/orm": "^2.9.4",
         "doctrine/persistence": "^2.5.2|^3.0",
         "phpdocumentor/reflection-docblock": "5.x-dev",
         "symfony/dependency-injection": "^5.4|^6.0",
         "symfony/form": "^5.4|^6.0",
+        "symfony/options-resolver": "^5.4|^6.0",
         "symfony/framework-bundle": "^5.4|^6.0",
         "symfony/phpunit-bridge": "^6.0",
         "symfony/property-info": "^5.4|^6.0",
         "symfony/security-csrf": "^5.4|^6.0",
         "symfony/twig-bundle": "^5.4|^6.0",
         "symfony/validator": "^5.4|^6.0",
+        "twig/twig": "^2.14.7|^3.0.4",
```

Comment:
I think this could actually be moved to `require`: there are a few places where we rely on `Twig` code directly. It's redundant in practice, as `ux-live-component` requires Twig, bug strictly speaking, I think this packages DOES directly need Twig.